server/routes/testutils/appSetup.ts

```diff
@@ -64,6 +63,5 @@ export function appWithAllRoutes({
   services?: Partial<Services>
   userSupplier?: () => Express.User
 }): Express {
-  govukOneLogin.default.authenticationMiddleware = () => (req, res, next) => next()
```

Comment:
Copied this from the way HMPPS Auth was set up but removing as it's actually redundant: we don't `app.use()` the auth so no need to do anything.